packages/runtime/src/templates/getApiHandler.ts

```diff
@@ -139,7 +139,7 @@ export const getApiHandler = ({
   const { config }  = require("${publishDir}/required-server-files.json")
   let staticManifest
   const path = require("path");
-  const pageRoot = path.resolve(path.join(__dirname, "${publishDir}", "serverless", "pages"));
+  const pageRoot = path.resolve(path.join(__dirname, "${publishDir}", "server"));
```

Comment:
So this is the new folder for serverless because it can be either coming from `app` or `pages`?

Reply:
Yeah, we need to use the parent because it could be both